CustomMetadata/SamlMetadataCustomConfiguration Example.json

```diff
@@ -0,0 +1,380 @@
+            {
+                "IsRequired": true,
+                "FriendlyName": "SpecVer",
+                "Name": "https://data.gov.dk/model/core/specVersion",
+                "NameFormat": ""
+            },
+            {
+                "IsRequired": true,
+                "FriendlyName": "NsisLevelOfAssurance",
+                "Name": "https://data.gov.dk/concept/core/nsis/loa",
+                "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri"
+            },
+            {
+                "IsRequired": true,
+                "FriendlyName": "Cvr",
+                "Name": "https://data.gov.dk/model/core/eid/professional/cvr",
+                "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri"
+            },
+            {
+                "IsRequired": true,
+                "FriendlyName": "OrganizationName",
+                "Name": "https://data.gov.dk/model/core/eid/professional/orgName",
+                "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri"
+            }
+        ],
+        "UseCustomOrganization": true,
+        "CustomOrganization": {
+            "DisplayNames": [
+                {
+                    "Name": "hhp@globeteam.com",
+                    "Language": "nl"
+                }
+            ],
+            "Names": [
+                {
+                    "Name": "nemlogin3",
+                    "Language": "da"
+                }
+            ],
+            "Urls": [
+                {
+                    "Uri": "http://hhpglobeteam.safewhere.eu",
+                    "Language": "fr"
+                }
+            ]
+        },
+        "UseCustomContactPerson": true,
+        "CustomContactPerson": {
+            "DisplayNames": [
+                {
+                    "Name": "hhp@globeteam.com",
+                    "Language": "nl"
+                }
+            ],
+            "Names": [
+                {
+                    "Name": "nemlogin3",
+                    "Language": "da"
+                }
+            ],
+            "Urls": [
+                {
+                    "Uri": "http://hhpglobeteam.safewhere.eu",
+                    "Language": "fr"
+                }
+            ]
+        },
+        "EncryptionAlgorithms": [
+            "http://www.w3.org/2001/04/xmlenc#aes128-cbc",
+            "http://www.w3.org/2001/04/xmlenc#aes256-cbc",
+            "http://www.w3.org/2009/xmlenc11#aes128-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes192-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes256-gcm"
+        ]
+    },
+    "kombit": {
+        "IncludeArtifactEndpoints": true,
+        "IncludeAuthenticationContextMethodClasses": true,
+        "IncludeContactPerson": true,
+        "NameIdentifierFormats": [
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"
+        ],
+        "IdentityProviderSupportedAttributes": [
```

Comment:
This example is wrong. There are a list of 5 or 6 required claims and each of them must be an element of the IdentityProviderSupportedAttributes array (similar to that of the nemlogin3 node). You can ask NGT for the list of claim.

Reply:
Fixed